README.md

```diff
@@ -66,28 +66,43 @@ ros2 run demo_nodes_cpp listener
 The listener node should start receiving messages over the `/chatter` topic.
 
 ## Configuration
-`rmw_zenoh` relies on separate configurations files to configure the Zenoh `router` and `session` respectively.
-To understand more about `routers` and `sessions`, see [Zenoh documentation](https://zenoh.io/docs/getting-started/deployment/).
+
+By default, `Zenoh sessions` created by `rmw_zenoh` will attempt to connect to a Zenoh router to receive discovery information.
+To understand more about `Zenoh routers` and `Zenoh sessions`, see [Zenoh documentation](https://zenoh.io/docs/getting-started/deployment/).
+
+### Checking for a Zenoh router.
+The `ZENOH_ROUTER_CHECK_ATTEMPTS` environment variable can be used to configure if and how a `Zenoh session` checks for the presence of a `Zenoh router`.
+The behavior is explained in the table below.
+
+
+| ZENOH_ROUTER_CHECK_ATTEMPTS |                                                 Session behavior                                                 |
+|:---------------------------:|:----------------------------------------------------------------------------------------------------------------:|
+|            unset            |                                                             Indefinitely waits for connection to a Zenoh router. |
+|             <= 0            |                                                                                        Skips Zenoh router check. |
```

Comment:
I'll suggest that we change this behavior slightly.  In particular, if it is unset or `0`, then it indefinitely waits, and if it is less than 0, we skip the check.  That feels a little more symmetric to me, and means that you can always set some value and get the behavior you want.

Reply:
Done https://github.com/ros2/rmw_zenoh/pull/135/commits/7904cee27097c8c0c6ed7110f09e4dff0285628c